src/sdk/cart/validate.ts

```diff
@@ -58,15 +58,42 @@ export const ValidateCartMutation = gql`
         name
       }
       gtin
+      additionalProperty {
+        propertyID
+        name
+        value
+        valueReference
+      }
     }
   }
 `
 
 export const isGift = (item: CartItem) => item.price === 0
 
+const serializeAdditionalProperty = (itemOffered: CartItem['itemOffered']) => {
+  if (
+    !itemOffered.additionalProperty ||
+    itemOffered.additionalProperty?.length === 0
+  ) {
+    return null
+  }
+
+  return itemOffered.additionalProperty
+    .map(({ propertyID }) => propertyID)
+    .join('-')
+}
+
 export const getItemId = (
   item: Pick<CartItem, 'itemOffered' | 'seller' | 'price'>
-) => `${item.itemOffered.sku}:${item.seller.identifier}:${item.price}`
+) =>
+  [
+    item.itemOffered.sku,
+    item.seller.identifier,
+    item.price,
+    serializeAdditionalProperty(item.itemOffered),
```

Comment:
A small nit for code styling, but, for me, it seems off to pass `itemOffered` for serializing `additionalProperty`. In my mind, it makes sense the pass additionalProperty to `serializeAdditionalProperty`. Also, I think `serializeAdditionalProperty`  can be inlined in here. The final proposition would be something like:
```suggestion
    item.itemOffered.additionalProperty?.map(({ propertyID }) => propertyID).join('-'),
```